### PR TITLE
Budgets new form

### PR DESCRIPTION
**Pull request by MariaCheca**
Where
=====
* **Related Issue:** https://github.com/consul/consul/issues/2277

What
====
- Removed `external_link` input in budget investment form. The database column will remain for previous budgets where the attribute was used, as it could be useful at some point, or just to keep all the original data as it was introduced.
- Changed `location` input label.

Screenshots
====
localhost_3000_budgets_1_investments_new laptop with mdpi screen

Test
====
As usual.

### CODE REVIEW

File: config/locales/es/budgets.yml [hunk 1]
```diff
@@ -58,6 +58,7 @@ es:
         map_location: "Ubicación en el mapa"
         map_location_instructions: "Navega por el mapa hasta la ubicación y coloca el marcador."
         map_remove_marker: "Eliminar el marcador"
+        location: "Información adicional de la localización"
```
Comment (bertocq) [inline config/locales/es/budgets.yml]: Maybe to be consistent with other spanish texts around locations the word **localización** should be changed for **ubicación** as that's what the map uses 3 times on title/subtitle/checkbox

Comment (MariaCheca) [reply config/locales/es/budgets.yml]: Good catch!

File: spec/features/budgets/investments_spec.rb [hunk 2]
```diff
@@ -311,7 +311,6 @@ def investments_order
       select  'Health: More hospitals', from: 'budget_investment_heading_id'
       fill_in 'budget_investment_title', with: 'Build a skyscraper'
       fill_in 'budget_investment_description', with: 'I want to live in a high tower over the clouds'
-      fill_in 'budget_investment_external_url', with: 'http://http://skyscraperpage.com/'
```
Comment (bertocq) [inline spec/features/budgets/investments_spec.rb]: I can see 3 more `budget_investment_external_url` occurences in the codebase 🤔  in two more tests:

**Comment by bertocq** [comment]
Maybe
https://github.com/consul/consul/blob/master/db/dev_seeds.rb#L441
and
https://github.com/consul/consul/blob/master/spec/factories.rb#L282
should be removed as well

To completly deprecate the usage of the attribute

File: app/views/budgets/investments/_investment_show.html.erb [hunk 3]
```diff
@@ -51,12 +51,6 @@
         </div>
       <% end %>
 
-      <% if investment.external_url.present? %>
```
Comment (bertocq) [inline app/views/budgets/investments/_investment_show.html.erb]: But... we want existing budgets with external_url's but no documents to still show those values right?

Comment (decabeza) [reply app/views/budgets/investments/_investment_show.html.erb]: Yes, we want to disallow this input field, but maintain the existing links 😌

File: spec/features/budgets/investments_spec.rb (hunk 2 above)
Comment (decabeza) [reply spec/features/budgets/investments_spec.rb]: Yes, in management panel also there is a new budget investment form with this field ☺️

**Pull request merged**